### DISCUSSION TITLE
Changes are coming for Rasp Pi OS

### DISCUSSION THREAD

**Discussion by Hartebee5t** [vicwomg/pikaraoke#271] [Ideas]
The follow-up Debian release post-Bullseye (called 'Bookworm') looks like there might be some tweaks needed for PiKaraoke, but the audio configs might be easier, eventually.
Here's the announcement article fro Rasp Pi HQ, describing how the Pi OS is being ported across:

https://www.raspberrypi.com/news/bookworm-the-new-version-of-raspberry-pi-os/?fbclid=IwAR0MFuhmBPDrjs6NxTC6HRYpUi_ggH3hhvbuqQve98EuiRZbNQacUBA590s

**Comment by BentWrench** [comment]
Getting ready to build this project from source. Do you think I should hold off on using Bookworm and use Buster instead?

**Reply by Hartebee5t** [reply]
I wouldn't be confident enough to say yes or no. I was merely bringing the article to the attention of those who would (such as the master, @vicwomg )

**Reply by BentWrench** [reply]
I ended up using the image and not building from source. Couldn't get it to work. Hopefully there will be a RP5 compatible update soon.
